### PR TITLE
Confirm that switching selection does not break the inspector.

### PR DESCRIPTION
**Pull request by seanparsons**
**Problem:**
We want some reassurance that the issue fixed in https://github.com/concrete-utopia/utopia/pull/2995 does not reoccur.

**Fix:**
Added a browser test that checks the selection results in the inspector being populated appropriately.

**Commit Details:**
- Added test that checks the inspector isn't broken by selection changing.

### CODE REVIEW

**Review by Rheeseyb — APPROVED** [review]
Beautiful!

**Review by balazsbajorics — APPROVED** [review]

File: editor/src/components/inspector/inspector.spec.browser2.tsx [hunk 1]
```diff
@@ -53,6 +126,46 @@ export var storyboard = (
...
+        'position-top-number-input',
+      ) as HTMLInputElement
+      expect(topField.value).toEqual(expectedTop)
+    }
+
+    // Click on the red div.
+    clickOnElementCheckTop('div-red', '50')
+
+    // Click on the green div.
+    clickOnElementCheckTop('div-red', '160')
```
Comment (enidemi) [inline editor/src/components/inspector/inspector.spec.browser2.tsx:164]:
```suggestion
    clickOnElementCheckTop('div-green', '160')
```

**Review by enidemi — APPROVED** [review]

File: editor/src/components/inspector/inspector.spec.browser2.tsx (hunk 1 above)
Comment (seanparsons) [reply editor/src/components/inspector/inspector.spec.browser2.tsx:164]: Hmmm, how did that not break?

Comment (seanparsons) [reply editor/src/components/inspector/inspector.spec.browser2.tsx:164]: Or it did!

Comment (seanparsons) [reply editor/src/components/inspector/inspector.spec.browser2.tsx:164]: OMG, it's the curse of `await` again...

Comment (seanparsons) [reply editor/src/components/inspector/inspector.spec.browser2.tsx]: Fixed in df66a9007.

**Pull request merged**